translations/config.json

```diff
@@ -88,8 +88,11 @@
         "nextcloud tasks",
         "nextcloud terms_of_service",
         "nextcloud text",
+        "nextcloud twofactor_gateway",
+        "nextcloud twofactor_nextcloud_notification",
         "nextcloud twofactor_totp",
         "nextcloud twofactor_u2f",
+        "nextcloud twofactor_webauthn",
```

Comment:
let's wait for https://github.com/nextcloud/twofactor_webauthn/pull/74

Reply:
and then also remove twofactor_u2f

Reply:
the app won't die yet. we still also need to ship an update with a warning and the documented migration path